packages/orb/commands/notify.yml

```diff
@@ -99,6 +99,11 @@ parameters:
     description: |
       Provide a URL to download a specific build of the Slack Orb CLI binary. This is useful if you are behind a firewall or need to test a specific build in your pipeline.
       This URL must be publicly accessible from the CircleCI build environment.
+  time_format:
+    type: string
+    default: ""
```

Comment:
Maybe it's better to set something here (e.g. `01/02/2006 15:04:05`) to show users how it's supposed to look like.

Reply:
Ya this is valid, it is the default anyway, but this will show it